Makefile.dep

```diff
@@ -784,6 +784,11 @@ ifneq (,$(filter uuid,$(USEMODULE)))
   USEMODULE += random
 endif
 
+# Enable periph_gpio when periph_gpio_irq is required
+ifneq (,$(filter periph_gpio_irq,$(USEMODULE)))
+  USEMODULE += periph_gpio
```

Comment:
In that case, should it not be `FEATURES_REQUIRED +=` ?
It has the same result of having the name put in `USEMODULE` but also triggers the `FEATURES` dependency checks.